src/infrastructure/api/index.ts

```diff
@@ -151,6 +163,50 @@ router.get(
   },
 );
 
+router.get('/tfl', (_req, res) => {
+  let secretsManager: SecretsManagerServiceInterface;
+  if (process.env.IS_OFFLINE === 'true') {
+    logger.debug('configuring local secret manager');
+    secretsManager = new LocalSecretsManagerService();
+  } else {
+    logger.debug('configuring aws secret manager');
+    secretsManager = new SecretsManagerService(new AWS.SecretsManager());
+  }
+  DatabaseService.build(secretsManager, mysql)
+    .then((dbService) => getTflFeedDetails(tflFeedQueryFunctionFactory, dbService))
+    .then((result) => {
+      const numberOfRows = result.length;
+      const fileName = `VOSA-${moment(Date.now()).format('YYYYMMDD')}-G1-${numberOfRows}-01-01.csv`;
+      logger.debug(`creating file for TFL feed called: ${fileName}`);
+      logger.info('Generating TFL File Data');
+      const processedResult = result.map((entry) => processTFLFeedData(entry));
+      const tflFeedProcessedData: string = processedResult
+        .map(
+          (entry) =>
+            `${entry.vrm_trm},${entry.vin},${entry.certificateNumber}, ${entry.modificationTypeUsed}, ${entry.testStatus}, ${entry.fuel_emission_id}, ${entry.createdAt}, ${entry.lastUpdatedAt}, ${entry.createdBy_Id}, ${entry.firstUseDate}`,
+        )
+        .join('\n');
+      logger.debug(`\nData captured for file generation: ${tflFeedProcessedData} \n\n`);
+      console.log(tflFeedProcessedData);
```

Comment:
Should replace console.log with a suitable logger method.

Reply:
Thanks for this spot, this is just a debug line I have missed and needs to be removed. Will do so now.